pkgs/tools/misc/pferd/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, aiohttp
+, beautifulsoup4
+, rich
+, keyring
+, certifi
+}:
+
+buildPythonApplication rec {
```

Please use `python3.pkgs.buildPythonApplication` and adjust the inputs. This allows to override dependencies.

```suggestion
python3.pkgs.buildPythonApplication rec {
```